airflow/providers/apache/hive/operators/hive_stats.py

```diff
@@ -100,15 +100,15 @@ def get_default_exprs(self, col: str, col_type: str) -> Dict[Any, Any]:
         if col in self.excluded_columns:
             return {}
         exp = {(col, 'non_null'): f"COUNT({col})"}
-        if col_type in ['double', 'int', 'bigint', 'float']:
+        if col_type in {'double', 'int', 'bigint', 'float'}:
```

Comment:
For short literal lists, converting to set is actually not a significant gain since creating a set is slower than list. (And tuple is probably faster.) But this is OK either way.